ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/config/tls/client/PicoCliClientTlsTrustOptions.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.config.tls.client;
+
+import static tech.pegasys.ethsigner.DefaultCommandValues.MANDATORY_FILE_FORMAT_HELP;
+
+import tech.pegasys.ethsigner.core.config.tls.client.ClientTlsTrustOptions;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
+import picocli.CommandLine.Option;
+
+class PicoCliClientTlsTrustOptions implements ClientTlsTrustOptions {
+  @Option(
+      names = "--downstream-http-tls-known-servers-file",
+      description =
+          "Path to a file containing the hostname, port and certificate fingerprints of web3 providers to trust.",
+      paramLabel = MANDATORY_FILE_FORMAT_HELP,
+      required = true,
+      arity = "1")
+  private Path knownServersFile;
+
+  @Option(
+      names = "--downstream-http-tls-invalidate-ca-signed",
```

Comment:
not sure on this name, generally a boolean value cli option would have enabled on the end. is the equivalent of the orion whitelist mode?

Reply:
This is the "enabled-ca" flag that we have discussed amongst us. This is probably an exception because we want CA to be enabled by default (which it is), and optionally disables it by specifying this flag. We are not using true/false value with it.

Reply:
maybe go with --downstream-http-tls-ignore-ca-signed?

Reply:
nit: is there another word we can use instead of invalidate? I.e. we're not saying that if you are CA authorised we'll reject your connection - just that you will not be authenticated against a CA ...
--downstream-http-tls-eschew-ca-auth (yeah, that maybe not great either)

Reply:
It was `disallow` in my initial attempt. `--downstream-http-tls-disallow-ca-signed`. Trent's suggestions on description to contain the word `validate` lead me to use `invalidate`.

Reply:
sorted.